entrypoint.sh

```diff
@@ -10,13 +10,15 @@ python3 /bin/scripts/generate_animation.py
 # Publish animation to `gh-pages` branch
 NAME=$(curl https://api.github.com/users/${GITHUB_ACTOR} | jq -r .name)
 ID=$(curl https://api.github.com/users/${GITHUB_ACTOR} | jq -r .id)
+BRANCH=${GITHUB_REF##*/}
 git config --global user.name "${NAME}"
 git config --global user.email "${ID}+${GITHUB_ACTOR}@users.noreply.github.com"
 git reset --hard
 git fetch
 git checkout gh-pages || git checkout -b gh-pages
 rm -rf $(ls -aI '.git') 2> /dev/null
```

Comment:
I didn't test this, but to me, it seems this line will delete the content of other branches in `gh-pages`. It should be something like:
```bash
[ -d ${BRACNH} ] && rm -rf ${BRANCH}/*
```

Reply:
Why not simply:
```bash
rm -rf ${BRANCH}
```

Reply:
That is much better :D